test/Microsoft.ML.OnnxTransformTest/OnnxTransformTests.cs

```diff
@@ -0,0 +1,215 @@
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.ImageAnalytics;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Runtime.RunTests;
+using Microsoft.ML.Runtime.Tools;
+using Microsoft.ML.Transforms;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+using System.Runtime.InteropServices;
+using Microsoft.ML.OnnxScoring;
```

Comment:
Ctrl + R+G

Reply:
Done

---
In reply to: [220397972](https://github.com/dotnet/machinelearning/pull/942#discussion_r220397972) [](ancestors = 220397972)